playground/backend/internal/fs_tool/fs_test.go

```diff
@@ -28,8 +28,8 @@ import (
 )
 
 const (
-	preparedModDir  = "testModDir"
-	preparedWorkDir = "workingDir"
+	sourceDir      = "sourceDir"
+	destinationDir = "destinationDir"
```

Comment:
why did we change prepared folder for Go to destination folder?

Reply:
I think because of a change in the logic of the CopyFiles function. Before that she only copied go.mod and go.sum but now also logging.properties. That is why Aydar changed the name of the folder to 'sourceDir'

Reply:
These `const` variables are used to test `CopyFile()` method, which required 3 parameters:
- `fileName`: name of the file which needs to copy.
- `sourceDir`: dir where the file is placed.
- `destinationDir`: dir where the new file will be placed.